MAINTAINERS.yml

```diff
@@ -211,9 +211,11 @@ Bluetooth:
     - subsys/bluetooth/audio/
     - include/zephyr/bluetooth/audio/
     - tests/bsim/bluetooth/audio/
+    - tests/bsim/bluetooth/ll/
```

Comment:
```suggestion
    - tests/bsim/bluetooth/host/
```

Reply:
@cvinayak note that this is the exclude section.

Reply:
@kruithofa please add a `tests/bsim/bluetooth/` in the files list above

Reply:
> @cvinayak note that this is the exclude section.

Ah... I did not notice that.